pkgs/data/fonts/inter/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, fetchzip }:
 
 let
-  version = "3.5";
+  version = "3.7";
 in fetchzip {
   name = "inter-${version}";
```

Comment:
Can we move it to ``pname``?

Reply:
I didn't think fetchzip recognized `pname`, I'll check now and yes if so then that sounds good.

As for fetchFromGitHub--that will work if the sources have the needed files but sometimes (esp re:fonts) there are downloads attached to a release that are basically "built" (although AFAIK they're whatever they want to put, but you know) and these can't be fetched with fetchFromGitHub.  Well at least AFAIK/in the past :).

Reply:
Update:

* fetchzip does not seem to support `pname` (nor does it accept `version` attribute, which also is a bit of a long-standing "quirk" motivating the `let version = "x.y.z"; in fetchzip ...` pattern.
* upstream's repo is indeed of the sort I describe, so no fetchFromGitHub here without also deciding to build the font ourselves which I'd prefer not to do in general especially for carefully managed build pipelines like they appear to use -- hard to replicate in a nix-y way and quite likely to end up with slightly different and probably inferior results.  However if someone wants to make this happen I'm not outright against it and would be happy to move to that after review.